tests_integ/single/test_basic_api.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+from parameterized import parameterized
+from tests_integ.helpers.base_test import BaseTest
+
+
+class TestBasicApi(BaseTest):
+    @parameterized.expand(
+        [
+            "basic_api_inline_openapi",
+            "basic_api_inline_swagger",
+            # "basic_api_with_cache", # no test case in java code base
```

Comment:
If we don't have a model for this and aren't making one, let's delete this.

Reply:
For sure.